### PR TITLE
[docker] disable arch until glibc is fixed.

### PR DESCRIPTION
**Pull request by rexhoffman**
## Motivation

Disable arch build until the glibc issue gets sorted:  https://bugs.archlinux.org/task/69563

### Have you read the [Contributing Guidelines on pull requests]

Y

## Test Plan

CI

## Related PRs

None

### CODE REVIEW

File: docker/ci/arch/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # while using circle we'll use circle's base image.
-FROM archlinux:20200908@sha256:7d1c7dd23c246895a34d6e8aa31eb29be573de2ce39b3e88315068e6d6cb5337 AS setup_ci_arch
+FROM archlinux:base-devel-20210207.0.15200@sha256:dc8477ed9937932806db8d120ab10983e00895779d0a0a982622edec3f772d6f AS setup_ci_arch
```
Comment (rexhoffman) [inline docker/ci/arch/Dockerfile:2]: There was a format change to arch that broke dependabot.

File: .github/workflows/ci-test.yml [hunk 2]
```diff
@@ -358,7 +358,7 @@ jobs:
           name: bench-baseline
           retention-days: 5
           path: |
-              /tmp/benches
+            /tmp/benches
```
Comment (rexhoffman) [inline .github/workflows/ci-test.yml:361]: auto format cleanup.

**Review by sausagee — APPROVED** [review]

**Pull request closed without merging**